nvflare/app_common/executors/launcher_executor.py

```diff
@@ -24,12 +24,15 @@
 from nvflare.apis.shareable import Shareable, make_reply
 from nvflare.apis.signal import Signal
 from nvflare.app_common.abstract.launcher import Launcher, LauncherCompleteStatus
+from nvflare.app_common.app_constant import AppConstants
 from nvflare.app_common.utils.fl_model_utils import FLModelUtils, ParamsConverter
 from nvflare.fuel.utils.pipe.pipe import Message, Pipe
 from nvflare.fuel.utils.pipe.pipe_handler import PipeHandler, Topic
 from nvflare.fuel.utils.validation_utils import check_object_type
 from nvflare.security.logging import secure_format_exception
 
+WAIT_TIME_AFTER_LAUNCHER_FINISH_FOR_GET_RESULT = 5.0
```

Comment:
changed, this is needed because
# LauncherExecutor need to wait additional time after the Lanucher finishes
# because it will takes some time to communicate the result
# (from external process sends and LauncherExecutor receives this last result)
# If we don't wait after the Lanucher finishes, then there is possibility
# that the result is still in transmission but we mark it as failed.

Reply:
Is it robust to hardcode this?